firewood/src/storage/buffer.rs

```diff
@@ -642,6 +642,23 @@ mod tests {
             .join("firewood")
     }
 
+    fn create_state_cache(
```

Comment:
nit: rename: `new_cached_space_for_test` would be better.

Nit's do not require fixing, so I'm approving at this point.